<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/splash.png</normaloff>:/splash.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 14px;
}

QWidget {
	color: white;
	background-color: black;
}

QLabel[style=encoderName] {
	qproperty-alignment: AlignCenter;
	border: 0px solid;
	color: black;
	background-color: white;
	padding: 2px 0px;
	max-height: 25px;
}

QLabel[style=encoderValue] {
	qproperty-alignment: AlignCenter;
	border: 0px solid;
	color: white;
	background-color: rgb(60, 60, 60);
	padding: 2px 0px;
	max-height: 25px;
	margin-top: -1px;
}

QProgressBar::chunk::vertical {
	height: 1 px;
}

QSlider {
	width: 40px;
}

QSlider::groove:vertical {
    background: transparent;
}

QSlider::handle:vertical {
    background: white;
	margin: -10px 0px;
	border-radius: 5px;
}

QSlider::add-page:vertical {
	margin: 0px 19px;
    background: white;
}

QSlider::sub-page:vertical {
	margin: 0px 19px;
    background: white;
}</string>
  </property>
  <property name="background" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="50,0,0,0,0,0" columnstretch="10">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="loading" native="true">
     <property name="minimumSize">
      <size>
       <width>790</width>
       <height>470</height>
      </size>
     </property>
     <widget class="QLabel" name="labelLogo">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>-5</y>
        <width>800</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../images.qrc">:/images/splash.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>325</y>
        <width>300</width>
        <height>10</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
	border: 1px solid white;
}

QProgressBar::chunk {
    background-color: white;
    width: 1px;
    margin: 0px;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="devMode" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>65</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="sliderEnc1">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderEnc2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderEnc3">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderEnc4">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderEnc5">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="patch" native="true">
     <property name="minimumSize">
      <size>
       <width>790</width>
       <height>470</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="100,0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="layoutPatch">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutInfos" stretch="0,0,0">
        <item>
         <widget class="SequencerWidget" name="sequencer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>505</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>00:00:09:264 s 4515.3 bar</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: darkred;
border: 0px;</string>
          </property>
          <property name="text">
           <string>EXIT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="layoutEncoders">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="labelEncValue1">
          <property name="text">
           <string>0</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderValue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelEncName1">
          <property name="text">
           <string>Encoder 1</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderName</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelEncName2">
          <property name="text">
           <string>Encoder 2</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderName</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="labelEncValue4">
          <property name="text">
           <string>0</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderValue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelEncName3">
          <property name="text">
           <string>Encoder 3</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderName</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelEncValue3">
          <property name="text">
           <string>0</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderValue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="labelEncName4">
          <property name="text">
           <string>Encoder 4</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderName</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelEncValue2">
          <property name="text">
           <string>0</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderValue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="labelEncName5">
          <property name="text">
           <string>Encoder 5</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderName</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="labelEncValue5">
          <property name="text">
           <string>0</string>
          </property>
          <property name="style" stdset="0">
           <string>encoderValue</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SequencerWidget</class>
   <extends>QWidget</extends>
   <header>ui/sequencerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
